ros_ign_bridge/CMakeLists.txt

```diff
@@ -28,14 +28,22 @@ if ("$ENV{IGNITION_VERSION}" STREQUAL "blueprint")
   set(IGN_MSGS_VER ${ignition-msgs4_VERSION_MAJOR})
```

Comment:
Line 22 needs to be updated: Default to Dome, support Citadel and Blueprint

Reply:
feff3eca0a13a4fdd841291e48385a7534fb117a